install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -179,11 +179,23 @@ following:
 ----
 /<example_fs> *(rw,root_squash)
 ----
-- The firewall must be configured to allow traffic to the mount point. The default port is 2049:
+- The firewall must be configured to allow traffic to the mount point:
 +
 ----
+NFSv4: The default port is 2049
```

Comment:
Can you make this part of the sentence leading in (before the code block)? I.e.:

```
The firewall must be configured to allow traffic to the mount point. For NFSv4, the default port is 2049:
```